Client/qtTeamTalk/preferencesdlg.cpp

```diff
@@ -625,8 +625,8 @@ void PreferencesDlg::slotTabChange(int index)
         connect(ui.spackBox, static_cast<void (QComboBox::*)(int)>(&QComboBox::currentIndexChanged), this, &PreferencesDlg::slotSPackChange);
         ui.sndVolSpinBox->setValue(ttSettings->value(SETTINGS_SOUNDEVENT_VOLUME, SETTINGS_SOUNDEVENT_VOLUME_DEFAULT).toInt());
         ui.sndeventPlaybackComboBox->addItem(tr("Default"), PLAYBACKMODE_DEFAULT);
-        ui.sndeventPlaybackComboBox->addItem(tr("One by One (TeamTalk Sound Device)"), PLAYBACKMODE_ONEBYONE);
-        ui.sndeventPlaybackComboBox->addItem(tr("Overlapping (TeamTalk Sound Device)"), PLAYBACKMODE_OVERLAPPING);
+        ui.sndeventPlaybackComboBox->addItem(tr("One by One"), PLAYBACKMODE_ONEBYONE);

```

Comment:
The reason I've written "(TeamTalk Sound Device)" is because it's played on your selected sound device. Unlike "Default" which is played on your OS selected sound device.

Reply:
Maybe the "Default" changed to "OS Selected".

Reply:
The default now uses also TeamTalk output device, so it's why I found this precision a bit useless.